internal/rootcoord/meta_table.go

```diff
@@ -54,7 +54,7 @@ const (
 	IndexMetaPrefix = ComponentPrefix + "/index"
 
 	// CollectionAliasMetaPrefix prefix for collection alias meta
-	CollectionAliasMetaPrefix = ComponentPrefix + "/collection-alias"
+	CollectionAliasMetaPrefix = ComponentPrefix + "/alias"
```

Comment:
Will there be any issue if any old version milvus upgrade to latest wit this prefix changed?

Reply:
Yes, old alias meta will not be able to load. According to the issue, the alias function is faulty now，so I guess few clients are using alias function.